controller/cache/cache.go

```diff
@@ -175,21 +175,23 @@ func NewLiveStateCache(
 	appInformer cache.SharedIndexInformer,
 	settingsMgr *settings.SettingsManager,
 	kubectl kube.Kubectl,
+	manifestGenTmpPath string,
```

Comment:
You don't have to pass this around, use the approach similar to what other options do above, e.g. `clusterCacheRetryUseBackoff`.